src/common/snippets/src/pass/align_element_type.cpp

```diff
@@ -50,11 +53,11 @@ bool ngraph::snippets::pass::AlignElementType::run_on_model(const std::shared_pt
     bool rewritten = false;
     auto ops = m->get_ordered_ops();
     for (auto& op : ops) {
-        if (is_in_out_op(op) || ov::is_type<ov::op::v0::Convert>(op)) {
+        if (is_in_op(op)) {
             continue;
         }
 
-        if (op_supports_only_exec_type(op)) {
+        if (is_executable_op_only_on_exec_type(op)) {
```

Comment:
Minor: It seems that to me that there are too many these tiny helper functions which affects code readability. Moreover, some of the helper functions are used only once. For example, you can substitute this condition with
`op_supports_only_exec_type(n) && !ov::is_type<ov::op::v0::Result>(n)` and can entirely eliminate `is_in_out_op()` and `is_executable_op_only_on_exec_type()`.

Reply:
Sounds better. Before these functions were needed but with new logic we can eliminate them. Thanks!